avail/distro/src/avail/Avail.avail/Foundation.avail/Foundation.avail

```diff
@@ -1042,6 +1042,11 @@ Extends
 		"character set lexer",
 		"style lexer"
 	),
+	"Libraries" =
```

Comment:
The module `Libraries` appears to be missing completely in this PR. I don't see the file anywhere.